crates/bevy_audio/src/audio_output.rs

```diff
@@ -136,8 +132,10 @@ pub(crate) fn play_queued_audio_system<Source: Asset + Decodable>(
                     );
                 }
 
+                // XXX scale seems jacked here.
```

Comment:
What is meant by this?

Reply:
No idea. I think this must be an artifact of the refactoring process. There doesn't seem to be an issue. Deleting.